### ISSUE TITLE
Regression: Downloaded jq has incorrect architecture

### ISSUE THREAD

**Issue by sovcharenko** [sanack/node-jq#638]
The latest release 4.2.0 downloads incorrect `jq` version (wrong platform) for Apple Silicone

```
% npm install node-jq@4.2.0

% node_modules/node-jq/bin/jq --version             
zsh: exec format error: node_modules/node-jq/bin/jq

% file node_modules/node-jq/bin/jq 
node_modules/node-jq/bin/jq: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, BuildID[sha1]=c30d429499cd0d0acd8f219ea623f0fed71b0099, for GNU/Linux 3.2.0, stripped
```

Previous version was downloading correct version:

```
%  npm install node-jq@4.0.1                    
%  node_modules/node-jq/bin/jq --version        
jq-1.6
%  file node_modules/node-jq/bin/jq                  
node_modules/node-jq/bin/jq: Mach-O 64-bit executable x86_64

```

**Comment by davesnx** [comment]
Hey @sovcharenko. Thanks for opening the issue, this has been introduced by https://github.com/sanack/node-jq/pull/625

**Comment by sovcharenko** [comment]
Interestingly, running install-binary downloads correct version

```
%  rm -f node_modules/node-jq/bin/jq; node node_modules/node-jq/scripts/install-binary.js 
Downloading jq from https://github.com/stedolan/jq/releases/download/jq-1.6/jq-osx-amd64
```

**Comment by davesnx** [comment]
1.6 looks wrong here

**Comment by sovcharenko** [comment]
Sorry, my bad pasted wrong output

```
rm -f node_modules/node-jq/bin/jq; node node_modules/node-jq/scripts/install-binary.js 
Downloading jq from https://github.com/jqlang/jq/releases/download/jq-1.7/jq-macos-arm64
Downloaded: 100%

```

**Comment by sovcharenko** [comment]
have a feeling that `jq` gets bundled into npm package. will check

**Comment by sovcharenko** [comment]
yep https://registry.npmjs.org/node-jq/-/node-jq-4.2.0.tgz has `jq` binary. and install-binary.js don't do anything as binary already exists

**Comment by sovcharenko** [comment]
not a semantic-release expert but feel like this is caused by #615

**Comment by sovcharenko** [comment]
yeah can confirm the issue is caused by

```
"bin": {
    "node-jq": "bin/jq"
}
```

with `"node-jq": "bin/jq"`

```
 %  npm publish           
npm WARN publish npm auto-corrected some errors in your package.json when publishing.  Please run "npm pkg fix" to address these errors.
npm WARN publish errors corrected:
npm WARN publish "repository.url" was normalized to "git+https://github.com/sanack/node-jq.git"
npm notice 
npm notice 📦  node-jq@0.0.0-semantic-release
npm notice === Tarball Contents === 
npm notice 1.1kB   LICENSE.md                       
npm notice 9.2kB   README.md                        
npm notice 808.0kB bin/jq                           
npm notice 48.5kB  docs/assets/logo-with-margin.png 
npm notice 2.2kB   package.json                     
npm notice 3.3kB   scripts/install-binary.js        
npm notice 64B     src/__test__/fixtures/1.js       
...
```

without `"node-jq": "bin/jq"`

```
 %  npm publish                                        
npm WARN publish npm auto-corrected some errors in your package.json when publishing.  Please run "npm pkg fix" to address these errors.
npm WARN publish errors corrected:
npm WARN publish "repository.url" was normalized to "git+https://github.com/sanack/node-jq.git"
npm notice 
npm notice 📦  node-jq@0.0.0-semantic-release
npm notice === Tarball Contents === 
npm notice 1.1kB   LICENSE.md                       
npm notice 9.2kB   README.md                        
npm notice 48.5kB  docs/assets/logo-with-margin.png 
npm notice 2.1kB   package.json                     
npm notice 3.3kB   scripts/install-binary.js        
npm notice 64B     src/__test__/fixtures/1.js       
npm notice 396B    src/__test__/fixtures/1.json     
npm notice 524.3kB src/__test__/fixtures/1.jsonl    
npm notice 21B     src/__test__/fixtures/2.json     
```

**Comment by sovcharenko** [comment]
Im afraid 4.2.1 has the same issue. as long as binary is referenced from package.json it will be packaged

**Comment by sovcharenko** [comment]
@davesnx `.npmignore` should fix the issue.

**Comment by sovcharenko** [comment]
Fixed in 4.2.2

**Comment by davesnx** [comment]
Hey @sovcharenko, haven't seen this issue updates while trying to fix our deploy process.

Thanks for being so good.
